demisto_sdk/common/constants.py

```diff
@@ -5,6 +5,10 @@
 class Errors:
     BACKWARDS = "Possible backwards compatibility break"
 
+    @staticmethod
+    def empty_display_configuration(file_path):
+        return "No display name were entered for {}.".format(file_path)
```

Comment:
Let's print the name of the display-missing parameter

Reply:
Done